### DISCUSSION TITLE
how ti minimize a opening window ?

### DISCUSSION THREAD

**Discussion by neronguyen** [prasanthrangan/hyprdots#1038] [Q&A] [answered]
thanks for answering me.

**Comment by kRHYME7** [comment]
A bare Window manager( tiling ) does not natively do that.

**Reply by neronguyen** [reply]
so I should move that app to special workspace with Super + Alt + S right ?

**Reply by kRHYME7** [reply, marked as answer]
See this one https://github.com/hyprwm/Hyprland/issues/995  People provides some ways to do it. 

Maybe close it or open another workspace?  Or make a script to handle those events (minimizing/ hiding windows  or move it to special workspace like you said. ). 
This is not a Full Desktop environment like gnome or kde.

**Reply by neronguyen** [reply]
thanks, i just new but really like it, so idomatic one
